.travis.yml

```diff
@@ -15,7 +15,7 @@ env:
     - PLONE_VERSION=5.2.x
     - PLONE_VERSION=5.1.x
 matrix:
-  allow_failures:
+  exclude:
```

Comment:
@ale-rt nice. I was looking for that but didn't recall the way to do this.

Reply:
Well, it was already like that on master...

Reply:
Jikes :/